blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -64,6 +64,7 @@ class RunDetails extends Component {
 
     componentWillMount() {
         this._fetchRun(this.props, true);
+        this.opener = locationService.previous;
```

Comment:
Good call, this is much simpler code. Can remove the `true` as second param up above though.